docs/decisions/0006_areas_of_responsibility.rst

```diff
@@ -0,0 +1,106 @@
+       for xAPI statements has not yet been created
+#. ``tutor-contrib-superset`` will:
+    #. Create an admin user
+    #. Create the Superset-specific MySQL database and permissions for the admin user (Note:
+       By default this uses the LMS MySQL instance to save on system resources)
+    #. Create a user in LMS for SSO
+    #. Register itself as an SSO target with the LMS
+    #. Disable local user management to force SSO login from LMS
+    #. Create a set of default roles and permissions based on LMS permissions for each user
+       (course staff, global staff, superuser, etc)
+    #. Create row-based filters for queries to enforce course-level permissions (Note:
+       since there are no datasets yet there is nothing to connect them to, but
+       they are created here as they would likely be useful for a non-OARS use case)
+    #. NOT create any ClickHouse databases, users, or tables
+
+The process of tying these plugins together is managed by the ``tutor-contrib-oars`` plugin, which
+does not provide a running service, but rather the opinionated configuration of the above services
+and commands to manage them:
+
+#. In ClickHouse:
+    #. Create any necessary databases (currently xAPI and course data)
+    #. Create and manage the schemas (currently this is a very basic proof-of-concept, but will
+       soon be managed via dbt)
+    #. Create users and permissions for Ralph, Superset, and CMS (for inserting course structure data)
+#. In Ralph:
+    #. Configure the ClickHouse user
+    #. Configure the ClickHouse database and xAPI statements table to write to
+    #. Optionally create test data in ClickHouse via Ralph
+#. In Superset:
+    #. Configure the ClickHouse user
+    #. Configure default dashboards, datasets, and reports
+    #. Attach the row level course filter to the appropriate datasets
+#. In LMS:
+    #. Add event-routing-backends as a Python requirement
+    #. Create an OARS user, necessary for the next step...
+    #. Configure event-routing-backends to use xAPI and send statements to Ralph
+       using the LMS user created in ``tutor-contrib-ralph``)
+
+The OARS plugin will also be the conduit for running dbt via Tutor "do" commands. The
+dbt integration will be covered in a separate ADR.
+
+Consequences
+************
+
+* The service-related plugins (ClickHouse, Ralph, Superset) can be run together or separately
+  with configurations different from the OARS defaults
+* A separate set of plugins could be created to replaces pieces of the stack with different
+  technologies (ex. replace Superset with Tableau, or ClickHouse with other supported Ralph
+  backend)
+* Extensions can be made at any level via additional Tutor plugins or other configuration
+* Configuration of the same services takes place in different plugins, which can be confusing.
+  The hope is that this ADR clarifies where to look and add new functionality.
```

Comment:
This is really good context and I wonder if it's worth being somewhat explicit about it in the initial context section of this document. Just a suggestion, not blocking.

Reply:
Good call, I added some more there